components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -61,8 +61,9 @@
         </div>
       </li>
     </ul>
+    <!-- Loading Success + Empty Services: Displayed when there are NO services from the selected filter -->
     <div class="no-services" *ngIf="(services$ | async)?.length === 0">
-      No available services with <b>{{ selectedHealth }}</b> status
+      No services with <b>{{ selectedHealth }}</b> status
```

Comment:
Could we make the message consistent with the table for empty service groups? 
None of the services returned {{ selectedHealth }}